### PR TITLE
quick update on this readme linking /github.com/Remi-Gau/COBIDAS_chckls

### PR DESCRIPTION
**Pull request by jbpoline**
to help folks knowing where they are :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,55 @@
...
+
+This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [neurovault website](https://neurovault.org/) when uploading your neuroimaging results.
+
+We are currently working to create a more complete checklist as described in the COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).
+
+## How to use it
+
+You can navigate each section on the left and then answer the questions corresponding to your fMRI analysis. This is meant to make sure that you have not forgotten any of the essential information in the methods and results parts of your article.
+
+At the end, you can click on `Export` (bottom left) to get a machine readable json file that recaps your method/results section: our next step is to use this to automate the part of the methods writing.
```
Comment (satra) [inline README.md]:
```suggestion
At the end, you can click on `Export` (bottom left) to get a zip file containing machine readable json files that captures information about your method/results section: our next step is to use this to automate the part of the methods writing and to submit the information alongside data submission to Neurovault and other data archives.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,55 @@
-# cobidas
+# eCOBIDAS
+
+**a checklist for methods and results reporting for neuroimaging studies**
+
+You can try this app [HERE](https://ohbm.github.io/cobidas/#/).
+
+This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [neurovault website](https://neurovault.org/) when uploading your neuroimaging results.
```
Comment (satra) [inline README.md]:
```suggestion
This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [Neurovault website](https://neurovault.org/) when uploading your neuroimaging results.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,55 @@
-# cobidas
+# eCOBIDAS
+
+**a checklist for methods and results reporting for neuroimaging studies**
```
Comment (satra) [inline README.md]:
```suggestion
**A checklist for methods and results reporting for neuroimaging studies**
```

**Comment by satra** [comment]
@Remi-Gau - i think you could put a pointer the google spreadsheet as a place where people can request to contribute. then they don't need to quite think about details of the schema, but focus on an easier route to add information to.

**Comment by Remi-Gau** [comment]
> @Remi-Gau - i think you could put a pointer the google spreadsheet as a place where people can request to contribute. then they don't need to quite think about details of the schema, but focus on an easier route to add information to.

Good idea!!!

Though given the size of that spreadsheet it might scare them too. :smile:

**Comment by Remi-Gau** [comment]
@satra 

Would it make sense for noa to re-use the same overall content of the README for the landing page of the app (i.e update `protocol/README-en.md`) ?

**Comment by satra** [comment]
> Would it make sense for noa to re-use the same overall content of the README for the landing page of the app (i.e update protocol/README-en.md) ?

you could even just point the `landingPage` key in the protocol to point to the main readme. no need to duplicate information.

**Comment by Remi-Gau** [comment]
> > Would it make sense for noa to re-use the same overall content of the README for the landing page of the app (i.e update protocol/README-en.md) ?
> 
> you could even just point the `landingPage` key in the protocol to point to the main readme. no need to duplicate information.

ah yes of course.

was afraid this would make for a very verbose landing page. :-)

File: README.md [hunk 4]
```diff
@@ -1,4 +1,83 @@
-# cobidas
+# eCOBIDAS
+
+**A checklist for methods and results reporting for neuroimaging studies**
+
+You can try this app [HERE](https://ohbm.github.io/cobidas/#/).
```
Comment (nicholst) [inline README.md]:
```suggestion
Try the app [HERE](https://ohbm.github.io/cobidas/#/)!
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,83 @@
...
+
+This checklist is still work in progress. The current prototype has been built using the 88 metadata items list used by the [Neurovault website](https://neurovault.org/) when uploading your neuroimaging results.
+
+We are currently working to create a more complete checklist as described in the COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).
```
Comment (nicholst) [inline README.md]:
```suggestion
We are currently working to create a more complete checklist as described in the OHBM COBIDAS reports for MRI, fMRI, EEG and MEG (see [below](#Background) for more information).
```

File: README.md [hunk 6]
```diff
@@ -1,4 +1,83 @@
...
+
+## Why this project?
+
+Poor methods and results description hinders the reproducibility and the replicability of research. It also makes it hard to compare new and old results and generally increases inefficiency in the research process. This project is built on the hope that improving methods and results reporting could improve our research.
+
+Follow the link if you want to know more about the [motivations behind this project](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/why_this_project.md).
+
+## Background
+
+This checklist is a project to make a user friendly checklist out the best practices [report](https://www.biorxiv.org/content/10.1101/054262v2) of the Committee on Best Practices in Data Analysis and Sharing ([COBIDAS](https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageid=3728)) of the Organization for human brain mapping.
```
Comment (nicholst) [inline README.md]:
```suggestion
This checklist is a project to make a user friendly checklist out the best practices [report](https://www.humanbrainmapping.org/COBIDASreport) of the Committee on Best Practices in Data Analysis and Sharing ([COBIDAS](https://www.humanbrainmapping.org/cobidas/)) of the [Organization for Human Brain Mapping](https://www.humanbrainmapping.org).
```

**Review by nicholst — COMMENTED** [review]
See a few tiny edits... @jbpoline I think you can easily merge these into your branch.  OTW, good to go!

**Comment by Remi-Gau** [comment]
Accepted the commits. Will merge in 24 hours unless I get a veto or further changes. :rocket:

**Comment by jbpoline** [comment]
Thanks for turning my simple pointer into the full fledge thing !

**Comment by Remi-Gau** [comment]
Mostly a copy paste for me. :+1:

File: README.md [hunk 7]
```diff
@@ -1,4 +1,83 @@
...
+
+From more frequent updates and behind the scenes, come and join us on the `cobidas_checklist` channel on the brainhack mattermost. <a href="https://mattermost.brainhack.org/brainhack/channels/cobidas_checklist"><img src="http://www.mattermost.org/wp-content/uploads/2016/03/logoHorizontal.png" width=100px /> Join our channel </a>
+
+There is also an [OSF project](https://osf.io/anvqy/) to centralize all the information and repos.
+
+We have a [list](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/goals.md) of short, middle, long term goals: if you are interested by any of those get in touch. Many of them do not necessarily require super-advanced technical skills (except maybe a certain love for working with spreadsheet and wanting them to be super organized) :wink:.
+
+## Contributors
+
+The list of the people involved in this project can be found [here](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/contributors.md#contributors).
```
Comment (Remi-Gau) [inline README.md]: fix broken link to contributors

```suggestion
The list of the people involved in this project can be found [here](https://github.com/Remi-Gau/COBIDAS_chckls/blob/master/contributions.md#whos-who-on-the-cobidas-checklist-team).
```

**Pull request merged**